src/Illuminate/Foundation/Testing/DatabaseMigrations.php

```diff
@@ -10,9 +10,13 @@
     public function runDatabaseMigrations()
     {
         $this->artisan('migrate');
+    }
 
-        $this->beforeApplicationDestroyed(function () {
-            $this->artisan('migrate:rollback');
-        });
+    /**
+     * @after
+     */
+    public function runDatabaseMigrateRefresh()
+    {
+        $this->artisan('migrate:refresh');
```

Comment:
Why not `migrate:reset` here, since the next test will run `migrate` during before test.